mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -141,8 +142,11 @@ def test_io_set_raw(fnames, tmpdir):
     shutil.copyfile(op.join(base_dir, 'test_raw.fdt'),
                     negative_latency_fname.replace('.set', '.fdt'))
     event_id = {eeg.event[0].type: 1}
-    pytest.raises(ValueError, read_raw_eeglab, montage=montage, preload=True,
-                  event_id=event_id, input_fname=negative_latency_fname)
```

Comment:
@larsoner this change as intended?

Reply:
I mean, this line *should* be a warning now, no longer an error, right?

Reply:
Yes, if the value is -1 -> warning, if lower -> error.